js/controller.js

```diff
@@ -953,7 +953,11 @@ OO.plugin('Html5Skin', function(OO, _, $, W) {
         this.setClosedCaptionsLanguage();
         this.removeBlur();
         this.state.isInitialPlay = false;
-        this.renderSkin();
+        this.renderSkin().then(() => {
+          if (this.state.cast.connected) {
```

Comment:
You need to know that this `cast` property on the state is only used on the sender side (web), the receiver will never have this  enabled or used at all, it depends on an event triggered by the ChromecastSender.js module